app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int = 7,
+                 clean_mark: int = 1,
+                 brand: str = "Unknown"
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float = 1,
+                 clean_power: float or int = 10,
+                 average_rating: float = 5,
+                 count_of_ratings: int = 1
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        cars_to_wash = [car for car in cars
+                        if car.clean_mark < self.clean_power]
+        for car in cars_to_wash:
+            income += self.calculate_washing_price(car)
```

Comment:
write `if car.clean_mark < self.clean_power` here, and delete 26-27 lines. it is better to use minimum `for` loops

Reply:
Got it